main.tf

```diff
@@ -76,6 +76,7 @@ resource "aws_elasticache_replication_group" "this" {
 # DNS
 ###############################################
 data "aws_route53_zone" "selected" {
```

Comment:
I'm thinking you don't need this anymore.

Reply:
:+1: done deleted it